python/agents/gemini-fullstack/.template/templateconfig.yaml

```
@@ -0,0 +1,37 @@
+# It defines the template's properties, settings, and deployment options.
+base_template: "adk_base"
+description: "A production-ready fullstack research agent that uses Gemini to strategize, research, and synthesize comprehensive reports with human-in-the-loop collaboration"
+example_question: "A report on the latest Google I/O event"
+settings:
+  deployment_targets: ["agent_engine", "cloud_run"]
+  tags: ["adk"]
+  commands:
+    override:
+      install: "uv sync --dev --extra jupyter && npm --prefix frontend install"
+    extra:
+      dev: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)